RELEASE_NOTES

```diff
@@ -0,0 +1,116 @@
+  embeddings (issue #1672).
+- mpv no longer freezes on wayland when the compositor stops asking it to draw
+  itself (e.g. when minimized) (issue #249).
+- `.ac3` files are no longer rejected by `--audio-file-auto` (issue #1759).
+- `ao_wasapi` now automatically enables `exclusive` when passthrough is
+  attempted (issue #1742).
+- Attempt to fix flickering on Intel VAAPI drivers (issue #1765).
+
+New features
+------------
+
+- `vo_opengl` features:
+  - Added `ewa_ginseng`, `ewa_hanning`, `robidoux`, `robidouxsharp`,
+    `oversample` and `haasnsoft` scalers.
+  - There are now `ewa_lanczonsoft` and `ewa_lanczossharp` aliases to
+    `ewa_lanczos` that are tuned to be blurrier and sharper, respectively.
+  - Added `gamma-auto` option that uses ambient light sensors to automatically
+    adjust the video gamma. See commit c028d78 for details.
+  - Added `blend-subtitles` option to draw subtitles directly into the video
+    instead of rendered afterwards. Potentially necessary for correct rendering
+    with files that use ASS subtitles for typesetting in combination with an
+    `icc-profile`. See warning on the manual.
+  - There is now a `tscale` option, used to choose the temporal scaler used
+    in the `interpolation` mode (previously `smoothmotion`).
+  - There is a new `scale-blur` parameter to adjust the amount of blur that
+    most of the filters produce. Deviating from the default may introduce
+    artifacts in EWA filters.
+- Add support to pitch correct stretched audio with librubberband.
+- Add support for the Raspberry Pi 2's hardware decoder when FFmpeg (or Libav)
+  is built with `--enable-mmal`. See commit 8fff125 for details.
+- The `--cache` option now accepts a "yes" option, that always enables a
+  `--cache-default`-sized cache on all cases a cache can be used, unless
+  `--cache-default` disables caching.
+- `ao_pulse`, `ao_coreaudio` and `ao_wasapi` now support device hotplugging.
+- New `--osd-align-x` and `--osd-align-y` options can be used to align the OSD
+  independently from subtitles.
+- New `--osd-bold` and `--sub-text-bold` options can be enabled to bold all
+  OSD or plain-text subtitle text, respectively.
+- Added a default keybind to the `u` key that enables/disables ASS style
+  overriding. When enabled, is equivalent to `--ass-style-override=force`.
+- There is now a `MOUSE_ENTER` keybind, that is called when the mouse cursor
+  enters the VO from outside.
+- The new `--ytdl-params` option can be set to arguments that are always given
+  to `youtube-dl` invocations. There is no sanity checking, so invalid options
+  can prevent `mpv` from working with http URLs.
+- `vo_direct3d_shaders` now supports NV12 colorspace without using stretchrect,
+  but it seems no drivers actually support that.
+- (Client API) Added a `rescan_external_files` command, as requested on issue
+  #1586.
+- (Client API) If enabled, initializing the Client API will now also load the
+  user's `mpv.conf`.
```

Comment:
If you're adding client API changes, maybe also mention that other API change... or link to the api changes file.